workflows/metadata.schema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "holohub/application/v1",
+  "type": "object",
+  "properties": {
+    "application": {
```

Comment:
workflow?

Reply:
You are right but for now, I will keep it as applications. It has more consequences in other part of the code that I need to take into account.

Reply:
Fixed